From 5827d9ccc393a93de4f4311eeae92b29b9b1d736 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?M=C3=A5rten=20Nordheim?= <marten.nordheim@qt.io>
Date: Mon, 15 Sep 2025 16:25:00 +0200
Subject: [PATCH 134/553] QTest: Add MSVC 2026 blacklist keyword

Pick-to: 6.9 6.8 6.5
Change-Id: Ia7c3cf6c52d9070ad85f647ddf383ca66eb3c8c3
Reviewed-by: Thiago Macieira <thiago.macieira@intel.com>
(cherry picked from commit e4241c9af0ac903a621adbe7a768cf1a2f23baa2)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/testlib/qtestblacklist.cpp | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/testlib/qtestblacklist.cpp b/src/testlib/qtestblacklist.cpp
index 7567e467469..22a132c9072 100644
--- a/src/testlib/qtestblacklist.cpp
+++ b/src/testlib/qtestblacklist.cpp
@@ -156,8 +156,10 @@ static QSet<QByteArray> keywords()
             << "msvc-2017"
 #  elif _MSC_VER <= 1929
             << "msvc-2019"
-#  else
+#  elif _MSC_VER <= 1949
             << "msvc-2022"
+#  else
+            << "msvc-2026"
 #  endif
 #endif
 
-- 
2.51.2

